pkgs/tools/security/ripasso/cursive.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkgconfig, ncurses, python3, openssl, libgpgerror, gpgme, xorg }:
+
+with rustPlatform;
+buildRustPackage rec {
+
+  version = "b64b1d9b697b2246b96db24a9d9ae571eab839aa";
+  name = "ripasso-cursive-${version}";
```

Comment:
Could use the pname attribute, that will append version to it automatically. So `pname = "ripasso-cursive";`